.github/workflows/go-releaser.yml

```diff
@@ -223,31 +223,33 @@ jobs:
       working-directory: artifacts
       run: zip -r octopus-cli.${{ needs.goreleaser.outputs.version }}.zip .
 
```

Comment:
Shouldn't we remove this?

Reply:
yep, forgot to delete it once I'd copied the behaviour 🙇‍♂️